client/packages/invoices/src/Returns/modals/CustomerReturn/useDraftCustomerReturnLines.ts

```diff
@@ -114,19 +114,17 @@ export const useDraftCustomerReturnLines = ({
 
     // TODO: error handling here
     // also need to consider what we do if the error was on the first page of the wizard
-    if (!returnId) {
-      await insert({
-        id: FnUtils.generateUUID(),
-        customerId,
-        outboundShipmentId,
-        customerReturnLines,
-      });
-    } else {
-      await updateLines({
-        customerReturnId: returnId,
-        customerReturnLines,
-      });
-    }
+    const returnOp = returnId ? await updateLines({
+      customerReturnId: returnId,
+      customerReturnLines,
+    }) : await insert({
+      id: FnUtils.generateUUID(),
+      customerId,
+      outboundShipmentId,
+      customerReturnLines,
+    });
+    return typeof returnOp === "number" ? returnOp : returnOp.invoiceNumber;
```

Comment:
awaiting update / insert and then converting to always return a number to be consumer focussed.